### PR TITLE
kurtosis assertoor tests

### PR DESCRIPTION
**Pull request by sudeepdino008**

### CODE REVIEW

**Comment by sudeepdino008** [comment]
the config.yaml needs to be downloaded off main (it currently points to this sudeep/kurtosis_assertoor branch)

**Comment by sudeepdino008** [comment]
@yperbasis we can merge this. The tests are working...There are more validator tests which might take a very long time (~48 hours), and I haven't enabled those.

Is there a way we get results of the ci runs on a discord channel or something? Basically I want to get reports of the results of the ci run. Right now, we have to go to github actions and check if things are working. It's better to automate that using discord hooks or emails.

File: erigon-lib/state/inverted_index.go
Comment (somnathb1) [inline erigon-lib/state/inverted_index.go:1]: Is this file relevant to this PR?

Comment (sudeepdino008) [reply erigon-lib/state/inverted_index.go:1]: no. Just a little refactoring

Comment (somnathb1) [reply erigon-lib/state/inverted_index.go:1]: Consider moving it to another PR, if it's just a chore and not related to this PR

Comment (yperbasis) [reply erigon-lib/state/inverted_index.go:1]: Yeah, it's not a big deal here, but in the future please avoid putting unrelated changes into the same PR.

**Review by yperbasis — APPROVED** [review]
